src/colinajose/model/people/Principal.java

```diff
@@ -0,0 +1,8 @@
+package colinajose.model.people;
+
+public class Principal extends Employee {
```

Comment:
Probably Principal and Secretary can just be a Type of employee not an actual class because they don't any additional information. Or if you want you can create only one class called Administrative and have one attribute called role which can be Principal or Secretary.